layouts/partials/home/results.html

```diff
@@ -9,11 +9,12 @@ <h2 class="border-b-2 border-green inline-block">Find an apprenticeship</h2>
     <div class="container">
       <div class="max-w-2xl mx-auto">
         <div id="hits">
+          <!-- Remove id #hits to test locally-->
           <div class="flex flex-wrap -mx-2 mt-0">
             {{ range .Site.Pages }}
             <a class="my-4 px-2 w-full md-w-1-2 lg-w-1-3" href="{{ .Params.link }}" target="_blank" rel="noopener noreferrer">
```

Comment:
Parcel will fail if we don't wrap this in an if statement checking for empty links.

We'll get a `Path must be a string. Received null` error due to four extra blank cards getting rendered by this loop. Those four cards have empty `href`s and Parcel does not seem to like those.

Reply:
Does the quote around null fix this or is that unrelated?

Reply:
I'm not sure anymore, but I don't see the issue in Netlify so I think we're good with this issue.